### DISCUSSION TITLE
Problems with commiting a folder

### DISCUSSION THREAD

**Discussion by Cript-blip** [community/community#39735] [New to GitHub]
Hi! 

I am new to git-hub and I am having some problems understanding how it works. Several weeks ago I integrated a folder in git-hub, but none of the changes I made were saved. Now, I am trying to do it commit them and push it to GitHub, but Git does not allow me to push the folder. Neither can I commit.  There is an image attached below, so maybe you can help me find a solution :)

![image](https://user-images.githubusercontent.com/62478794/203291399-f257ebea-2018-4432-a9fb-7cc4d0dbddbd.png)

PD: Sorry if I dont use correctly some of the github terms

**Comment by ossanna16** [comment]
Hi there @Cript-blip and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by Cript-blip** [reply]
Thanks!!

**Comment by Preetiraj3697** [comment]
@Cript-blip unfortunately this is a restriction enforced by the GitHub server.

If the commits are still unpublished, you should be able to undo them (using the button below the commit form) to bring back that file which exceeds this limit into your working directory. If you avoid committing it again, you should be able to push the other changes fine.

If you still want to have this large file in version control, Desktop supports Git LFS - [here's how to configure it for your repository](https://git-lfs.github.com/). Once your repository is Git LFS-enabled, Desktop will use that when pushing in the future.

Please also follow along with https://github.com/desktop/desktop/issues/997 which is our exploration of warning the user before committing a file that will trigger this warning.

**Reply by Cript-blip** [reply]
Thank you very much. I did what you said in the first paragraph!!
